libtransmission/session.cc

```diff
@@ -341,6 +341,16 @@ std::optional<std::string> tr_session::WebMediator::bind_address_V6() const
     return std::nullopt;
 }
 
+bool tr_session::WebMediator::has_source_address_V4() const
+{
+    return !!session_->global_source_address(TR_AF_INET);
```

Comment:
nit: use `std::optional<T>::has_value()` for the sake of readability

Reply:
3354fa8afc14206b42a76a7e1e3fc15628da3c3f